File: apps/vinculacion/app/models/vinculacion/empleado.rb
module Vinculacion
  class Empleado < ActiveRecord::Base

    self.table_name = "rh_empleados"

    #belongs_to :departamento
    #belongs_to :sede

    def nombre_completo
      "#{nombre} #{apellido_paterno} #{apellido_materno}"
    end

    def avatar_url
      username = email.split(/@/)[0]
      "http://cimav.edu.mx/foto/#{username}/64"
    end

  end
end

File: apps/proyectos/app/assets/javascripts/proyectos/models/empleado.js
App.Empleado = DS.Model.extend({
  nombre:            DS.attr('string'),
  apellido_paterno:  DS.attr('string'),
  apellido_materno:  DS.attr('string'),
  email:             DS.attr('string'),
  telefono:          DS.attr('string'),
  nombre_completo:   DS.attr('string'),
  avatar_url:        DS.attr('string'),

  departamento: DS.belongsTo('departamento')
});